CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+* PR title must follow convention: `<type>(<scope>): <subject>`.
+
+For example:
+  ```
+   feat(replication): support for enabling replication
+   ^--^ ^-----^   ^-----------------------^
+     |     |         |
+     |     |         +-> PR subject, summary of the changes
+     |     |
+     |     +-> scope of the PR, i.e. component of the project this PR is intend to update
+     |
+     +-> type of the PR.
+  ```
+  Most common types are:
+    * `feat`        - for new features, not a new feature for build script
+    * `fix`         - for bug fixes or improvements, not a fix for build script
+    * `chore`       - changes not related to production code
+    * `docs`        - changes related to documentation
+    * `style`       - formatting, missing semi colons, linting fix etc; no significant production code changes
+    * `test`        - adding missing tests, refactoring tests; no production code change
+    * `refactor`    - refactoring production code, eg. renaming a variable or function name, there should not be any significant production code changes
+    * `cherry-pick` - if PR is merged in master branch and raised to release branc
+
+## Code Reviews
+All submissions, including submissions by project members, require review. We use GitHub pull requests for this purpose. Consult [GitHub Help](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests) for more information on using pull requests.
+
+* If your PR is not getting reviewed or you need a specific person to review it, please reach out to the OpenEBS Contributors. See [OpenEBS Community](https://github.com/openebs/openebs/tree/master/community)
+
+* If PR is fixing any issues from [github-issues](github.com/openebs/istgt/issues) then you need to mention the issue number with link in PR description. like : _fixes https://github.com/openebs/istgt/issues/56_
+
+* If PR is for bug-fix and release branch(like v1.9.x) is created then cherry-pick for the same PR needs to be created against the release branch. Maintainer of the Project needs to make sure that all the bug fix after RC release are cherry-picked to release branch.
+
+### For maintainers
+* We are using labelling for PR to track it more effectively. Following are valid labels for the PR.
+   - **Bug** - if PR is a **bug to existing feature**
+   - **Enhancement** - if PR is a **feature request**
+   - **Maintenance**  - if PR is not related to production code. **build, document or test related PR falls into this category**
+   - **Documentation** - if PR is about **tracking the documentation work for the feature**. This label should not be applied to the PR fixing bug in documentations.
+
+* We are using following label for PR work-flow:
+   - **Hold-Merge** - if PR needs to complete few activities.
+   - **Hold-Review** - if PR is under Work In Progress, not yet covered all the scenarios.
+   - **Release-note** - if PR required to mention in release note.
```

Comment:
not required now, because we are having `changelog`  folder for changelog. So we can skip this label.

Reply:
Done